common/beerocks/bwl/dwpal/base_wlan_hal_dwpal.h

```diff
@@ -73,8 +73,8 @@ class base_wlan_hal_dwpal : public virtual base_wlan_hal,
      * @param max_buffer_size buffer size
      * @return size of returned data (success if > 0)
      */
-    size_t dwpal_nl_cmd_get(const std::string &ifname, unsigned int nl_cmd,
-                            unsigned char *out_buffer, const size_t max_buffer_size);
+    ssize_t dwpal_nl_cmd_get(const std::string &ifname, unsigned int nl_cmd,
```

Comment:
Changed the code in `mon_wlan_hal_dwpal` to match the new signature
I didn't test the error handling, but since the change was from  `size_t ` to `ssize_t` they should function identically